core/server/models/user.js

```diff
@@ -293,6 +301,8 @@ User = ghostBookshelf.Model.extend({
 
         if (status === 'active') {
             query.query('whereIn', 'status', activeStates);
+        } else if (status === 'inactive') {
```

Comment:
fixture utils is doing `User.findOne({status: inactive})` - this all happens automatically. 
https://github.com/TryGhost/Ghost/pull/7545/commits/12d00d429d668ee4a9f6320fe9dbda9cf8bfcb7f#diff-a34b3b1dabb543b82a1fe58ebee85abbR321

so it never found the owner and crashed. then i added this fix, worked then.
any idea how to solve differently?